### PR TITLE
Delete unused files

### PR DESCRIPTION
**Pull request by madisonscott**

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -30,12 +30,9 @@ Redirect 302 /smugmug https://cmubuggy.smugmug.com/
 RedirectMatch 302 (?i)/RD20 https://cmu.zoom.us/j/91610341394
 
 #top level sections
-RewriteRule		^buzz[/]*$						index.php?s=buzz				[NC,L]
```
Comment (madisonscott) [inline .htaccess:33]: A live chat that is no longer needed because of Discord

File: .htaccess [hunk 2]
```diff
@@ -30,12 +30,9 @@ Redirect 302 /smugmug https://cmubuggy.smugmug.com/
...
 RewriteRule		^history[/]*$					index.php?s=history				[NC,L]
 RewriteRule		^raceday[/]*$					index.php?s=raceday				[NC,L]
 RewriteRule		^search[/]*$					index.php?s=search				[NC,L]
-RewriteRule		^seniors[/]*$					index.php?s=seniors				[NC,L]
```
Comment (madisonscott) [inline .htaccess:37]: Replaced this with a WordPress page

File: .htaccess [hunk 3]
```diff
@@ -30,12 +30,9 @@ Redirect 302 /smugmug https://cmubuggy.smugmug.com/
...
-RewriteRule		^store[/]*$						index.php?s=store				[NC,L]
```
Comment (madisonscott) [inline .htaccess:38]: This was blank. Also I imagine if we set up a store, we will not be writing any custom code.

File: content/live.inc [hunk 4]
```diff
@@ -1,25 +0,0 @@
-<nav class="navbar navbar-light navbar-expand bg-light mb-3">
```
Comment (madisonscott) [inline content/live.inc:1]: We use WordPress for this page

File: email/200903.html [hunk 5]
```diff
@@ -1,105 +0,0 @@
-<html>
```
Comment (madisonscott) [inline email/200903.html:1]: I put all these files in an Archive folder in our Google Drive

File: index.php [hunk 6]
```diff
@@ -9,50 +9,20 @@
     $s = $_GET["s"];
   }
   $title = "CMU Buggy Alumni Association";
-  $headline = "Records: 2:02.16, 2:23.27";
```
Comment (madisonscott) [inline index.php:12]: Appears to be unused everywhere

File: scripts/heats2011.csv [hunk 7]
```diff
@@ -1,18 +0,0 @@
-"racedayid";"raceclassid";"heat";"lane1";"lane2";"lane3"
```
Comment (madisonscott) [inline scripts/heats2011.csv:1]: Sent these to Jeremy

File: scripts/update_reddit_cache.php [hunk 8]
```diff
@@ -1,18 +0,0 @@
-<?php
```
Comment (madisonscott) [inline scripts/update_reddit_cache.php:1]: Confirmed these are not set up as jobs on the server

File: .htaccess (hunk 2 above)
Comment (robsiemb) [reply .htaccess:37]: Is this in place?  http://cmubuggy.org/news/seniors seems to be a 404.

File: index.php (hunk 6 above)
Comment (robsiemb) [reply index.php:12]: Yeah, I just noticed this.  If so, we should remove all of them, not just some (e.g. history case further down in index.php, as well as the uses in gallery/themes/cmubuggy/views/page.html.php and news/wp-content/themnese/cmubuggy-child/header.php)

**Review by robsiemb — APPROVED** [review]
Yeah, this mostly looks safe except for the issue with seniors/

File: email/200903.html (hunk 5 above)
Comment (robsiemb) [reply email/200903.html:1]: Only concern is that they might be actual links in existing emails (like "click here if this doesn't render in your email reader" type stuff).  The images might be important too.

Of course, they're 10+ years old, so... not super relevant.

**Pull request merged**